src/troubleshooting/errors.rst

```diff
@@ -0,0 +1,118 @@
+
+.. code-block:: bash
+
+  (2023-04-20 21:37:10): [be[example.com]] [generic_ext_search_handler] (0x0020): [RID#1434] sdap_get_generic_ext_recv failed: [110]: Connection timed out [ldap_search_timeout]
+  (2023-04-20 21:37:16): [be[example.com]] [sssd_async_connect_done] (0x0020): [RID#1434] connect failed [110][Connection timed out].
+  (2023-04-20 21:37:16): [be[example.com]] [sssd_async_connect_done] (0x0020): [RID#1434] connect failed [110][Connection timed out].
+  (2023-04-20 21:37:16): [be[example.com]] [sssd_async_socket_init_done] (0x0020): [RID#1434] sdap_async_sys_connect request failed: [110]: Connection timed out [ldap_network_timeout].
+  
+In the above example ``sdap_get_generic_ext_recv failed`` is getting failed due to the network issue we could see ``Connection timed out [ldap_network_timeout]`` and ``connection timed out [ldap_search_timeout]``. As a workaround consider increasing the value of ``ldap_network_timeout`` and ``ldap_search_timeout``. But need to identify the underlying network issue.
+
+
+krb5_auth_timeout
+******************
+
+.. code-block:: bash
+
+  (Fri Apr 14 16:37:19 2023) [sssd[be[example.com]]] [krb5_child_timeout] (0x0040): Timeout for child [23514] reached. In case KDC is distant or network is slow you may consider increasing value of krb5_auth_timeout.
+  
+The above error is self-explanatory when trying to connect to the LDAP server, but the LDAP server is responding very slowly due to some reason. One of them could be an issue with the firewall or a slow network. As a workaround, consider increasing the value of ``krb5_auth_timeout`` which is 6 seconds by default.
+
+TSIG error with server: tsig verify failure
+********************************************
+
+.. code-block:: bash
+
+  (Fri Apr 14 16:37:19 2023) example.com sssd[87108]: ; TSIG error with server: tsig verify failure
+
+When SSSD is configured with id_provider = ad, by default, sssd will try to update the DNS record using the nsupdate command (the default value of the dyndns_update option is true). If, for some reason, tsig/nsupdate is failing, sssd will return ``TSIG error with server: tsig verify failure``. This error can be safely ignored if ``Dynamic DNS update`` is not being used, or if  ``Dynamic DNS update`` can be disabled by using ``dyndns_update = false`` under the ``[$domain]`` section of the ``/etc/sssd/sssd.conf`` file. If ``Dynamic DNS update`` is being used, then in order to identify the issue, enable sssd debugging using https://sssd.io/troubleshooting/basics.html.
```

Comment:
`or if  ``Dynamic DNS update`` can be disabled by using `

or if ``Dynamic DNS update`` is disabled using...